llvm/lib/Support/YAMLTraits.cpp

```diff
@@ -39,6 +39,10 @@ IO::IO(void *Context) : Ctxt(Context) {}
 
 IO::~IO() = default;
 
+SourceMgr& IO::getSourceMgr() {
```

Comment:
This feels like an unrelated changes or change that could be done in isolation first

Reply:
Sure, I can break this up.